### PR TITLE
Feature/migrate to vuepress

### PR DESCRIPTION
**Pull request by Dragonesis**
Finish

### CODE REVIEW

File: .vuepress/src/components/ProjectLogo/ProjectLogo.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <svg class="icon" width="41.534px" height="45.558px"><use xlink:href="#logo-icon"></use></svg>
+    <svg class="text" width="68.829px" height="26.945px"><use xlink:href="#logo-text"></use></svg>
+  </router-link>
+</template>
+
+<script>
+export default {
+  name: 'ProjectLogo',
+  props: {
+    color: 'color' | 'white'
```
Comment (stcherenkov) [inline .vuepress/src/components/ProjectLogo/ProjectLogo.vue]: Did you mean `'colored'`?

Comment (stcherenkov) [reply .vuepress/src/components/ProjectLogo/ProjectLogo.vue]: And also property name, smth like `theme` maybe?

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +0,0 @@
-FROM node:8-onbuild AS build
```
Comment (stcherenkov) [inline Dockerfile]: These removed files are removed temporary?

File: news/news2.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Al Hilal Bank executes the world&#x27;s first Blockchain Sukuk transaction
+date: Nov. 26, 2019
+description: Abu Dhabi Al Hilal Bank is the first Islamic bank in the world to leverage Blockchain technology for the resale and settlement of an Islamic Sukuk. Blockchain was used to transact a secondary market deal...
+img: /assets/img/content/cover/zawya.png
+img2x: /assets/img/content/cover/zawya@2x.png
+---
+
+<NewsPage>
```
Comment (stcherenkov) [inline news/news2.md]: Let's remove this wrapper. We already put this file under `/news` – so it is not something else, for sure.

**Comment by stcherenkov** [comment]
Ok, looks nice, but for full assessment needs an example of i18n – both for news-like pages and for landing-like pages.

File: Dockerfile (hunk 2 above)
Comment (Dragonesis) [reply Dockerfile]: Yes

File: .vuepress/src/components/FooterMenu/FooterMenu.vue [hunk 4]
```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class='footer-menu'>
+    <div class='box' data-aos='fade-down' data-aos-duration='300' data-aos-delay="150">
+      <div class='title'>Company</div>
+      <div class='item '>
+        <a href='about.html' class='link ' >
```
Comment (IViol) [inline .vuepress/src/components/FooterMenu/FooterMenu.vue]: I assume, that now we can remove `.html` from links

File: .vuepress/theme/Layout.vue [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  methods: {
+    getColor() {
+      switch (this.$route.path) {
+        case '/':
+          return 'white';
+      default:
+        return 'color';
+      }
+    },
+    getTypePgae() {
```
Comment (IViol) [inline .vuepress/theme/Layout.vue]: getTypePgae -> getTypePage

Comment (IViol) [reply .vuepress/theme/Layout.vue]: is it ok to place this file within `.vuepress` dir?

**Review by IViol — COMMENTED** [review]
- please setup/update linters
- some of necessary for deployment files were removed, need to re-check them

File: .vuepress/theme/Layout.vue (hunk 5 above)
Comment (Dragonesis) [reply .vuepress/theme/Layout.vue]: This folder is assumed in VuePress itself

File: careers/ux-analyst-and-designer.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+published: true
+title: UX Analyst & Designer
+description: Jibrel is looking for a Middle Python Developer to join its Saint-Petersburg office. Successful candidate will develop backend applications that interact with the different blockchains.
```
Comment (IViol) [inline careers/ux-analyst-and-designer.md:4]: wrong description

**Review by stcherenkov — APPROVED** [review]

**Review by flexxnn — APPROVED** [review]

**Review by Creepypoke — APPROVED** [review]

**Pull request merged**